addons/project/static/src/views/project_task_kanban/project_task_kanban_record.js

```diff
@@ -21,4 +24,113 @@ export class ProjectTaskRecord extends Record {
         }
         return context;
     }
+
+    modifyName(changes, new_values, element) {
```

Comment:
as i will stop working on the task, i will mention the remaining points to do (in case it will be continued by me or anyone in the team in the future),
the task is done, we just need to:
- delete modifyName in this file
- delete modifyOtherFields in this file and in hr_timesheet
- add 
```suggestion
      {
        '~': (param, changes, errors) => {
            if (!isNaN(param)
                this.addShortcutError(`~${param}`, errors);
            else {
            // adapt this.addOption() to do the following behavior when we want to cumulate sum
                    if (!('planned_hours' in changes))
                            changes['planned_hours'] = 0;

                     changes['planned_hours'] += +param
            }
        },
```
- write a UT
cc: @xavierbol